### ISSUE TITLE
Model checkpoints

### ISSUE THREAD

**Issue by halikhani** [YuanJianhao508/RAG-Driver#10]
Hi, very interesting work! I was wondering when we can access model checkpoints. Thank you !

**Comment by YuanJianhao508** [comment]
Hi @halikhani , thank you for your interest in our work! We are currently cleaning and gathering everything to release and will update you once it is ready!

**Comment by halikhani** [comment]
Thanks a lot! actually I was able to fine-tune the model, but there is a big gap between the results I got and the results provided in the paper. What kind of scaling did you use since as far as I know, the range for metrics like METEOR is 0-1 and for CIDEr is 0-10. I assume all the numbers in your evaluation section for these metrics are multiplied by 100 from the original metrics. Please let me know if my assumption is correct. Thank you!

**Comment by YuanJianhao508** [comment]
Hi @halikhani , thank you for asking! Yes, for the metric we scale it by 100 just for the purpose for demonstrating the result. Thank you very much!

**Comment by abhigoku10** [comment]
@halikhani  can u please share ur trained model for testing of the inference on google drive or onedrive it would be helpful @YuanJianhao508  when can we expect the pretrained models
